include/deal.II/lac/trilinos_tpetra_sparse_matrix.templates.h

```diff
@@ -140,7 +140,7 @@ namespace LinearAlgebra
 
 
     // reinit_matrix():
-    namespace
+    namespace SparseMatrixImpl
```

Comment:
That's the right thing to do regardless as we don't want anonymous namespaces in header files.